setup.py

```diff
@@ -53,7 +53,7 @@
 
 from setuptools import setup
 
-_PACKAGE_NAME = os.environ.get("PACKAGE_NAME", "")
+_PACKAGE_NAME = ""
```

Comment:
added by mistake

Reply:
Yes, the annoyance of the `pip install -e .` ;) I need to un-commit those files.

Reply:
added again by mistake :D

Reply:
@tchaton please revert this before merging